assets/component-mega-menu.css

```diff
@@ -1,15 +1,17 @@
 .mega-menu {
-  position: initial;
+  position: static;
 }
 
-.mega-menu-content {
+.mega-menu__content {
   background-color: rgb(var(--color-background));
   border-left: 0;
   border-radius: 0;
   border-right: 0;  
   left: 0;
-  padding-bottom: 3.2rem;
-  padding-top: 3.2rem;
+  max-height: calc(var(--viewport-height, 100vh) - var(--header-bottom-position, 200px));
```

Comment:
Or we could leave a bit of space at the bottom to click on 🤷

Reply:
Added a `4rem` bottom offset https://github.com/Shopify/dawn/pull/1338/commits/2f854a4217aa903c4e9d258b82193fd5bf63ae73. This offset isn't really rendered in iOS Safari due to [this behaviour](https://github.com/Shopify/dawn/pull/1338#discussion_r813078382), but since the menu can always be closed by clicking on the `summary` element, we don't necessarily need alternative ways to trigger close.